recipes/PeoplesSpeech/ASR/transformer/hparams/conformer_large.yaml

```diff
@@ -0,0 +1,284 @@
+# ############################################################################
+# Model: E2E ASR with Transformer
+# Encoder: Conformer Encoder
+# Decoder: Transformer Decoder + (CTC/ATT joint) beamsearch + TransformerLM
+# Tokens: BPE
+# losses: CTC + KLdiv (Label Smoothing loss)
+# Training: People's Speech 28,000 hours
+# Authors:  Titouan Parcollet
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 3407
+__set_seed: !apply:speechbrain.utils.seed_everything [!ref <seed>]
+output_folder: !ref results/conformer_transducer_large/<seed>
+output_wer_folder: !ref <output_folder>/
+save_folder: !ref <output_folder>/save
+train_csv: !ref <save_folder>/train.csv
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files
+hf_download_folder: !PLACEHOLDER
+subsets: ["clean", "dirty"] # _sa sets are removed.
```

Comment:
wait. this is for training right? 

What about evals? I think one should document somewhere that you must DL the evals splits as well :p